src/modules/universal-discovery/components/content-meta-preview/content.meta.preview.component.tsx

```diff
@@ -1,9 +1,29 @@
-import React, { Component, Fragment } from 'react';
-import PropTypes from 'prop-types';
+import * as React from 'react';
+import { Component, Fragment } from 'react';
```

Comment:
Imports for React library in TS work a little bit differently. 
`import React, { Component, Fragment } from 'react';` generates error. 

There are some discussions on the Internet, e.g.: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/5128. Nonetheless it seems that the TS import is correct, and the current one is possible because of how Babel works. Maybe we can force old import by some tsconfig.json options.

But, AFAIK, most React projects just `import * as React from 'react';` and then use `React.Component`, `React.Fragment` etc.